src/bin/doodle/format/base.rs

```diff
@@ -1,5 +1,5 @@
 use doodle::byte_set::ByteSet;
-use doodle::{Expr, Format, FormatModule, FormatRef, Pattern};
+use doodle::{ Expr, Format, FormatModule, FormatRef, Pattern };
```

Comment:
More unexpected formatting changes here 🤔

Reply:
Indeed. Not sure if reverting this change and re-running `rustfmt` makes a difference, however.